runtime/interpreter/value.go

```diff
@@ -1026,7 +1077,7 @@ func (v *StringValue) Transfer(
 }
 
 func (v *StringValue) Clone(_ *Interpreter) Value {
-	return NewStringValue(v.Str)
+	return NewUnmeteredStringValue(v.Str)
```

Comment:
Why is this unmetered?

Reply:
`Clone` is an internal function that is not used by the interpreter. It can e.g. be used for a state migration